itest/rust/src/object_tests/virtual_methods_test.rs

```diff
@@ -75,6 +75,11 @@ impl Node2DVirtual for ReadyVirtualTest {
     fn ready(&mut self) {
         self.implementation_value += 1;
     }
+
+    #[cfg(any())]
+    fn to_string(&self) -> GodotString {
+        "Removed by #[cfg]".into()
```

Comment:
Could also be `panic!`, in case it's ever compiled in and called?

Or maybe even [`compile_error!`](https://doc.rust-lang.org/std/macro.compile_error.html), if that works here? (Similar thoughs apply below).

Reply:
Great idea, done!